.github/workflows/handle-stale-discussions.yml

```diff
@@ -1,41 +1,15 @@
 name: HandleStaleDiscussions
 on:
   schedule:
-    - cron: '*/10 * * * *'
-  discussion:
-    types:
-      - created
-      - edited
-      - category_changed
-      - locked
-      - answered
-  discussion_comment:
-    types:
-      - created
+    - cron: '0 */4 * * *'
```

Comment:
@peterwoodworth , I added the event triggers for taking immediate actions on the discussions. . As per initial discussions with Tom, it was agreed to run workflow every 10-15 mins. Reducing it to every 6 hrs might not be a good idea.

Reply:
This should only need to run every few hours or so. The critical part to do regularly is to process for new proposed answers so that we can post the instructions text ASAP. This can now be done immediately whenever a new answer is proposed if the workflow is triggered by a new discussion comment.